examples/repo_example/basic_repo.py

```diff
@@ -117,7 +118,7 @@ def _in(days):
 local_path = Path(__file__).resolve()
 target_path = f"{local_path.parts[-2]}/{local_path.parts[-1]}"
 
-target_file_info = TargetFile.from_file(target_path, local_path)
+target_file_info = TargetFile.from_file(target_path, str(local_path))
```

Comment:
Was this also flagged by mypy?

(Also it is odd, because I remember already casting this to `str` at some point because `TargetFile.from_file` raised an error with local_path being a `Path` object. But I can't find it in #1685. I wonder were it went... Anyways, good that you/mypy fixed it.)

Reply:
Yes, `mypy` noticed that.
Yet another argument to why we do need linting quickly as possible on new code.